pmda/density.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import MDAnalysis as mda
+import time
+from MDAnalysis.lib.util import fixedwidth_bins
+from MDAnalysis.analysis.density import Density
+from .parallel import ParallelAnalysisBase
+
+class DensityAnalysis(ParallelAnalysisBase):
+    """Parallel density analysis.
+
+    Parameters
+    ----------
+    universe : MDAnalysis.Universe
```

Comment:
You don't actually use a `universe`. Instead you hand over the atomgroup that you want to analyze.
```reST
atomgroup : AtomGroup
    Group of atoms (such as all the water oxygen atoms) that are being analyzed.
    If this is an updating AtomGroup then you need to set `atomselection` and 
    and also ``updating=True`` (this is a workaround until PMDA can natively deal
    with `dynamic selections<https://www.mdanalysis.org/docs/documentation_pages/selections.html#dynamic-selections>`_).
```